src/main/connector/system/Windows/Windows.yaml

```diff
@@ -0,0 +1,210 @@
+        fileSystemInformation:
+          type: wmi
+          namespace: root\CIMv2
+          query: SELECT DeviceID,FreeSpace,Size,VolumeName FROM Win32_LogicalDisk
+          computes:
+          - type: subtract
+            column: 3
+            value: $2
+      mapping:
+        source: ${source::monitors.fileSystem.simple.sources.fileSystemInformation}
+        attributes:
+          id: $1
+          system.filesystem.device: $1
+          system.filesystem.volumeName: $4
+        metrics:
+          system.filesystem.usage{system.filesystem.state=free}: $2
+          system.filesystem.usage{system.filesystem.state=used}: $3
+  service:
+    simple:
+      sources:
+        # Name;ProcessId;StartMode;State
+        serviceInformation:
+          type: wmi
+          namespace: root\CIMv2
+          query: SELECT Name,ProcessId,State FROM Win32_Service WHERE StartMode != 'Disabled'
+      mapping:
+        source: ${source::monitors.service.simple.sources.serviceInformation}
+        attributes:
+          id: $1
+          processId: $2
+        metrics:
+          system.service.state: $3
+  pageFile:
+    simple:
+      sources:
+        # Name;CurrentUsage;CurrentUsage
+        pageFileUsageInformation:
+          type: wmi
+          namespace: root\CIMv2
+          query: SELECT Name,CurrentUsage,CurrentUsage FROM Win32_PageFileUsage
+        # Name;MaximumSize
+        pageFileSettingsInformation:
+          type: wmi
+          namespace: root\CIMv2
+          query: SELECT Name,MaximumSize FROM Win32_PageFileSetting
+        #Name;CurrentUsage;Utilization;Name;MaximumSize
+        pageFileCombinedInformation:
+          type: tableJoin
+          leftTable: ${source::monitors.pageFile.simple.sources.pageFileUsageInformation}
+          rightTable: ${source::monitors.pageFile.simple.sources.pageFileSettingsInformation}
+          leftKeyColumn: 1
+          rightKeyColumn: 1
+          computes:
+          - type: divide
+            column: 3
+            value: $5
+      mapping:
+        source: ${source::monitors.pageFile.simple.sources.pageFileCombinedInformation}
+        attributes:
+          id: $1
+        metrics:
+          system.paging.usage: $2
+          system.paging.utilization: $3
+  network:
+    simple:
+      sources:
+        # Name;PacketsOutboundDiscarded;PacketsReceivedDiscarded;PacketsSentPersec;PacketsReceivedPersec;PacketsOutboundErrors;PacketsReceivedErrors
+        networkInformation:
+          type: wmi
+          namespace: root\CIMv2
+          query: >
+            SELECT Name,PacketsOutboundDiscarded,PacketsReceivedDiscarded,PacketsSentPersec,PacketsReceivedPersec,PacketsOutboundErrors,PacketsReceivedErrors
+            FROM Win32_PerfFormattedData_Tcpip_NetworkInterface
+      mapping:
+        source: ${source::monitors.network.simple.sources.networkInformation}
+        attributes:
+          id: $1
+        metrics:
+          system.network.dropped{network.io.direction="transmit"}: $2

```

Comment:
Please add `system.network.io` metric (from [Otel spec](https://opentelemetry.io/docs/specs/semconv/system/system-metrics/#metric-systemnetworkio)) in bytes.